db/migrate/20170207173837_set_service_display_and_retired_to_false.rb

```diff
@@ -0,0 +1,12 @@
+class SetServiceDisplayAndRetiredToFalse < ActiveRecord::Migration[5.0]
+  Service = Class.new(ActiveRecord::Base)
```

Comment:
Oy vey. I wish there was an intuitive way to catch this, but I'll just go ahead and add that in